### PR TITLE
EO-NOTASK - Fixed width for library body.

### PR DESCRIPTION
**Pull request by mihaelad**
Fixed width for body bibliotek

#### Checklist

- [x] My complies with [our coding guidelines](../docs/code_guidelines.md).
- [x] My code passes our static analysis suite. If not then I have added a comment explaining why this change should be exempt from the code standards and process.
- [x] My code passes our continuous integration process. If not then I have added a comment explaining why this change should be exempt from the code standards and process.

### CODE REVIEW

File: themes/ddbasic/sass/components/panel/class-panel.scss [hunk 1]
```diff
@@ -391,7 +391,7 @@
 
 .pane-node-field-ding-library-lead,
 .pane-node-field-ding-library-body {
-  @include span-columns(5 of 8);
+  width: 100%;
```
Comment (guddo) [inline themes/ddbasic/sass/components/panel/class-panel.scss:394]: Shouldn't this be `@include span-columns(8 of 8);` instead of width?

Comment (mihaelad) [reply themes/ddbasic/sass/components/panel/class-panel.scss:394]: `@include span-columns(8); ` is  the same `width: 100%`

Comment (guddo) [reply themes/ddbasic/sass/components/panel/class-panel.scss:394]: Yes, but why change the logic of usage?

**Comment by guddo** [comment]
@mihaelad Please undo all unneeded changes, as we need to keep the diff to origin as slim as possible.

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 553a2ea189d9f50d8efe5c23db07d3fa47259e55
  * Commits must be in the following format: [issue ID] - [short summary of change]

Guidelines are available at https://github.com/easySuite/ding2

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 487969dd96183d9a02f37d4649cc8ebc4f8129c8
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 003d40a496e8c5ed4ffcb9edf2e5a645d065c698
  * Commits must be in the following format: [issue ID] - [short summary of change]

Guidelines are available at https://github.com/easySuite/ding2

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 487969dd96183d9a02f37d4649cc8ebc4f8129c8
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 003d40a496e8c5ed4ffcb9edf2e5a645d065c698
  * Commits must be in the following format: [issue ID] - [short summary of change]

Guidelines are available at https://github.com/easySuite/ding2

---
This message was auto-generated by https://gitcop.com

**Review by guddo — APPROVED** [review]

**Pull request merged**